### DISCUSSION TITLE
How long does it take to get your benefits after you've been verified?

### DISCUSSION THREAD

**Discussion by SwiftDust** [community/community#57688] [GitHub Education] [answered]
### Hi everyone,

I've been verified yesterday evening (within the hour, kudos to GitHub!) but after that, I never heard anything of them. On the verification page a panel shows me that I've been verified, but I never got a mail of them. I also didn't get my benefits yet. Before I verified, a message stated that **verification** could take up to 4 days , but it seems very odd to me that the GitHub staff manually has to grant me the benefits (as far as I know, a computer verifies me). So, how long does it take to get my benefits?

Regards,
Mart
Student

**Comment by saiparthiv** [comment]
I got verified on 6th June and 4 days later I still didn't receive any benefits.

**Reply by SwiftDust** [reply]
Still not right now? I found 72 hours somewhere so I might open a support ticket if I still not got 'em in 72 hours.

**Reply by saiparthiv** [reply, marked as answer]
I got verified just now. It took them 4 1/2 days.

**Reply by NafiRahim** [reply]
@saiparthiv I'm now been waiting for 3 days. yet no sign of benefit. Did they send an email about the student pack benefit being delivered or something??

**Reply by micahgabbard** [reply]
Yes it took lil over 3 1/2 days for me to be fully verified. Thank you @github

**Reply by NafiRahim** [reply]
It took 3 days and 2 hr for me.

**Reply by micahgabbard** [reply]
I am finally verified but I dont have even copilot access/benefit? is there something wrong with the system or must i do more for the full benefits? @github @admin

**Comment by guang-xiao03** [comment]
On Wednesday, June 14, 2023, I successfully completed the verification process for my email address. However, I have not been able to use the GitHub Student Developer Pack yet.

**Reply by SwiftDust** [reply]
Yeah, I've been verified over approx. 3 days so the best thing to do is just wait 4 days and if they still didn't give you access just open a support ticket

**Reply by LiAo365** [reply]
Me too, I successfully completed the verification process at the same day, until now, I have not been able to use the GitHub Student Developer Pack yet.

**Reply by micahgabbard** [reply]
I have been verified for 2 days now and no benefits either. wonder if its a backlog kind of thing or just malformed code XD

**Reply by micahgabbard** [reply]
Yeah idky but i still dont have student developer pack or copilot access as a student either.

**Reply by NafiRahim** [reply]
@micahgabbard  copilot comes within the student pack. for me, it took 3 days and a few hours. if you are verified wait a little long. you will receive it soon.

**Reply by micah4232** [reply]
I have been verified already but now i dont see stundet pack or copilot. I been verified over weeks now and nothing

**Reply by NafiRahim** [reply]
@micah4232 I remembered a pop-up after applying that they will reply to you within 10 days. and during the semester start time, it took more time. Hence you are a verified student no need to worry. soon you will get a mail regarding the student pack available to your account.

**Comment by Kovium** [comment]
Does this always need so much time to verify, till the students get the access to the benefits? (-.-)

**Comment by Kovium** [comment]
I have no idea.... Im still waiting too -.-

________________________________
Von: Nafi Rashid Rahim ***@***.***>
Gesendet: Mittwoch, 28. Juni 2023 21:03
An: community/community ***@***.***>
Cc: Michel Kowal ***@***.***>; Comment ***@***.***>
Betreff: Re: [community/community] How long does it take to get your benefits after you've been verified? (Discussion #57688)


@saiparthiv<https://github.com/saiparthiv> I'm now been waiting for 3 days. yet no sign of benefit. Did they send an email about the student pack benefit being delivered or something??

**Reply by NafiRahim** [reply]
I got my benefit at midnight. took 75 hr almost.

**Comment by kanishk11426** [comment]
My identitiy has been verified but I am not able to get to use github copilot
![Screenshot 2024-09-19 005845](https://github.com/user-attachments/assets/ad2f4629-e977-429e-90d9-02951c5e06e5)
